build/mvn

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
only bad thing about this is it won't work if you aren't running a local nifi...I'd prefer to leave out since ideally we'd be building this with a build script. i'll think about it